cmd/zoekt-webserver/main.go

```diff
@@ -409,6 +411,26 @@ func (s *loggedSearcher) Search(ctx context.Context, q query.Q, opts *zoekt.Sear
 	return sr, err
 }
 
+func (s *loggedSearcher) StreamSearch(ctx context.Context, q query.Q, opts *zoekt.SearchOptions, sender stream.Sender) error {
+	stats := struct {
+		sync.Mutex
+		*zoekt.Stats
```

Comment:
nit: I don't think this needs to be a pointer. Out of interest why are you using an struct? Is it so you can do `stats.Lock()`? I expect just doing `var mu sync.Mutex` is more idiomatic?

Reply:
> nit: I don't think this needs to be a pointer
👍 

> Is it so you can do stats.Lock()
Yes, I think the struct is a nice way to bind the mutex to whatever it protects. It reads nice and seeing stats.Lock() in the code is more obvious than just mu.Lock().